config/src/config.rs

```diff
@@ -276,6 +281,17 @@ pub struct Mining {
     pub enabled: bool,
 }
 
+/// Ntp-related configuration
+#[derive(PartialStruct, Debug, Clone, PartialEq)]
+#[partial_struct(derive(Deserialize, Default, Debug, Clone, PartialEq))]
+pub struct Ntp {
+    /// Period that indicate the validity of a ntp timestamp
+    pub update_period: i64,
+
+    /// Server to query ntp information
+    pub server: String,
```

Comment:
Even if it is not implemented, the idea is to support multiple NTP servers. We can make this option an array and just use the first one for the moment.

```suggestion
    pub servers: Vec<String>,
```